VideoLocker/res/drawable/row_discussion_first_comment_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/edx_grayscale_neutral_white" />
+    <stroke
+        android:width="@dimen/edx_hairline"
+        android:color="@color/edx_grayscale_neutral_light" />
+    <corners android:radius="@dimen/edx_box_radius"
```

Comment:
The global radius is not needed since we are defining them at an individual level.